plasmapy/formulary/frequencies.py

```diff
@@ -96,17 +99,18 @@ def gyrofrequency(B: u.T, particle: ParticleLike, signed=False, Z=None) -> u.rad
     gyration around magnetic field lines and is given by:
 
     .. math::
-        ω_{c} = \frac{Z e B}{m}
+        ω_{c} = \frac{\|Z\| e B}{m}
```

Comment:
Technically whether the abs is there depends on whether `signed` is set, right?

Reply:
Good catch!